resources/elasticsearch/mappings/logs.js

```diff
@@ -0,0 +1,19 @@
+// Specific syntax:
+// https://serverless.com/framework/docs/providers/aws/guide/variables/#reference-variables-in-javascript-files
```

Comment:
I think you can remove this comment now, as you've improved the previous syntax which was ignored in the linter.